css/styles.css

```diff
@@ -0,0 +1,137 @@
+body {
+    margin: 0;
+}
+.topnav {
+    background-color:rgb(25, 50, 126);
+    overflow: hidden;
+    width: 100%;
+    height: 3em;
+}
+
+.topnav a {
+    float: left;
+    color: whitesmoke;
+    text-align: center;
+    padding-top: 1vh;
+    padding-right: 2vw;
+    text-decoration: none;
+    font-size: 1.2em;
+    text-shadow: 2px 2px black;
+}
+.topnav img {
+    float: left;
+    margin-top: 0.4vh;
+    margin-right: 0.5vw;
+    margin-left: 0.2vw;
+    width: 2em;
+    height: 4vh;
+}
+
+.topnav p {
+    float: left;
+    color: whitesmoke;
+    font-size: 2em;
+    margin-top: 0.4vh;
+}
+
+.topnav-right {
+    float: right; 
+}
+
+.twitter-filter-section p {
+    margin-top: 2.2vh;
+    margin-left: 8vw;
+    font-size: 1.2em;
+    float: left;
+    margin-right: 0.5vw;
+}
+
+.twitter-body .twitter-filter-section input {
+    margin-top: 2.2vh;
+}
+
+.twitter-dashboard-left {
+    width: 75%;
+    float: left;
+}
+
+.twitter-flex-container-dashboard {
+    display: flex;
+    align-content: flex-start;
```

Comment:
Q: what is the difference between:  align-content, align-items and justify-content?

Reply:
align-content used to align the flex lines 
align-items used to align the item vertically
justify-content used to align the item horizontally

Reply:
לא מדויק

Reply:
The align-content used to align the flex lines.